apps/src/sites/studio/pages/programming_environments/edit.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import getScriptData from '@cdo/apps/util/getScriptData';
+import ProgrammingEnvironmentEditor from '@cdo/apps/lib/levelbuilder/code-docs-editor/ProgrammingEnvironmentEditor';
+import ExpandableImageDialog from '@cdo/apps/templates/lessonOverview/ExpandableImageDialog';
+import instructionsDialog from '@cdo/apps/redux/instructionsDialog';
+import {getStore, registerReducers} from '@cdo/apps/redux';
+import {Provider} from 'react-redux';
+
+$(document).ready(() => {
+  registerReducers({
+    instructionsDialog
```

Comment:
why do you need instructions dialog here?

Reply:
Oh the ExpandableImageDialog. That name is definitely confusing for what its doing

Reply:
Yeah agreed. I'll add a comment